packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { useComments } from '@/api/hooks'
+import { CommentOrderByInput } from '@/api/queries'
+import { EmptyFallback } from '@/components/EmptyFallback'
+import { Text } from '@/components/Text'
+import { Comment } from '@/components/_comments/Comment'
+import { Select } from '@/components/_inputs/Select'
+import { absoluteRoutes } from '@/config/routes'
+import { COMMENTS_SORT_OPTIONS } from '@/config/sorting'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import { useUser } from '@/providers/user'
+
+import { CommentWrapper, CommentsSectionHeader, CommentsSectionWrapper } from './VideoView.styles'
+
+type CommentsSectionProps = {
+  disabled?: boolean
+  videoAuthorId?: string
+}
+
+export const CommentsSection: React.FC<CommentsSectionProps> = ({ disabled, videoAuthorId }) => {
+  const [sortCommentsBy, setSortCommentsBy] = useState(CommentOrderByInput.ReactionsCountDesc)
+  const { id } = useParams()
+  const { activeMemberId } = useUser()
+  const { comments, loading } = useComments(
+    { where: { video: { id_eq: id } }, orderBy: sortCommentsBy },
+    { skip: disabled || !id }
+  )
+  const mdMatch = useMediaMatch('md')
+  const placeholderItems = loading && !comments ? Array.from({ length: 4 }, () => ({ id: undefined })) : []
+
+  const handleSorting = (value?: CommentOrderByInput | null) => {
+    if (value) {
+      setSortCommentsBy(value)
+    }
+  }
+
+  if (disabled) {
+    return (
+      <CommentsSectionWrapper>
+        <EmptyFallback title="Comments are disabled" subtitle="Author has disabled comments for this video" />
+      </CommentsSectionWrapper>
+    )
+  }
+
+  return (
+    <CommentsSectionWrapper>
+      <CommentsSectionHeader>
+        <Text variant="h400">
+          {loading || (comments && !comments.length) ? 'Comments' : `${comments && comments.length} comments`}
+        </Text>
+        <Select
+          size="small"
+          labelPosition="left"
+          label={mdMatch ? 'Sort by' : ''}
+          value={sortCommentsBy}
+          items={COMMENTS_SORT_OPTIONS}
+          onChange={handleSorting}
+          disabled={loading}
+        />
+      </CommentsSectionHeader>
```

Comment:
You didn't add a divider below the header.

![image](https://user-images.githubusercontent.com/51168865/165507739-9470c422-700a-4d56-9b4d-f61a26350c39.png)